source/guides/rapids-docker-with-cuda.md

```diff
@@ -0,0 +1,371 @@
+| CUDA Compatibility | `cuda-compat` | Forward compatibility libraries for older drivers |
+
+### Runtime Components (Required for RAPIDS on pip)
+
+The **runtime** images include all base components plus additional CUDA libraries:
+
+| Component                     | Package Name     | Purpose                                            |
+| ----------------------------- | ---------------- | -------------------------------------------------- |
+| **All Base Components**       | (see above)      | Core CUDA runtime                                  |
+| CUDA Libraries                | `cuda-libraries` | Comprehensive CUDA library collection              |
+| CUDA Math Libraries           | `libcublas`      | Basic Linear Algebra Subprograms (BLAS)            |
+| NVIDIA Performance Primitives | `libnpp`         | Image, signal & video processing primitives        |
+| Sparse Matrix Library         | `libcusparse`    | Sparse matrix operations                           |
+| Profiling Tools               | `cuda-nvtx`      | NVIDIA Tools Extension for profiling               |
+| Communication Library         | `libnccl2`       | Multi-GPU and multi-node collective communications |
+
+### Development Components (Optional)
+
+The **devel** images add development tools to runtime images:
+
+- CUDA development headers and static libraries
+- CUDA compiler (`nvcc`)
+- Debugger and profiler tools
+- Additional development utilities
+
+```{note}
+Development components are typically not needed for RAPIDS usage unless you plan to compile CUDA code within your container.
+```
+
+## Getting the Right Components for Your Setup
+
+The [NVIDIA CUDA Container Images repository](https://gitlab.com/nvidia/container-images/cuda) contains a `dist/` directory with pre-built Dockerfiles organized by CUDA version, Linux distribution, and container type (base, runtime, devel).
+
+### Supported Distributions
+
+**Debian/Ubuntu-based (uses `apt`):**
+
+- **Ubuntu**: `ubuntu2004/`, `ubuntu2204/`, `ubuntu2404/`
+
+**RHEL/CentOS-based (uses `yum`/`dnf`):**
+
+- **Rocky Linux**: `rockylinux8/`, `rockylinux9/`, `rockylinux10/`
+- **RHEL/CentOS**: `ubi8/`, `ubi9/`, `ubi10/`
+- **Amazon Linux**: `amzn2023/`
+- **Azure Linux**: `azl3/`
+- **OpenSUSE**: `opensuse15/` (CUDA 13.0+ only)
+
+### Key Differences by Distribution Type
+
+**Ubuntu/Debian distributions:**
+
+- Use `apt-get install` commands
+- Repository setup uses GPG keys and `.list` files
+
+**RHEL/CentOS/Rocky Linux distributions:**
+
+- Use `yum install` or `dnf install` commands
+- Repository setup uses `.repo` configuration files
+- Include repository files: `cuda.repo-x86_64`, `cuda.repo-arm64`
+
+### Installing CUDA components on your container
+
+1. Navigate to `dist/{cuda_version}/{your_os}/base/` or `runtime/` in the repository
+2. Open the `Dockerfile` for your target distribution
+3. Copy all `ENV` variables for package versions and NVIDIA Container Toolkit support
```

Comment:
When looking at a Dockerfile I'm not sure I follow what do you mean by copy all `ENV` variables for package versions and NVIDIA Container Toolkit support. 

I see there are examples below, if there is no other way of making this clearer at this step, maybe mention that there are examples below.